src/request/sign-transaction/SignTransaction.css

```diff
@@ -1,154 +1,300 @@
-    font-size: 2.25rem;
-    line-height: 2.5rem;
-    height: 2.5rem;
-    font-weight: 600;
-    margin-bottom: 1.25rem;
+.identicon.clip {
+    position: relative;
 }
 
-.transaction .account > .address {
-    white-space: unset;
+.identicon.clip > img {
+    clip-path: url(#nimiq-rounded-hexagon);
 }
 
-.transaction .arrow {
-    margin: 3rem 1rem;
-    height: 2rem;
-    width: 2rem;
+.identicon.clip:after {
+    content: ' ';
+    display: block;
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: 100%;
+    background-size: contain;
+    background-image: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" fill="none" width="146" height="146"><path d="M113.829 16.20974L142.47 65.7903c2.578 4.461 2.578 9.9584 0 14.4194L113.829 129.79c-2.57 4.461-7.335 7.21-12.488 7.21H44.0555c-5.1529 0-9.918-2.749-12.4877-7.21L2.92725 80.2097c-2.56966698-4.461-2.56966698-9.9584 0-14.4194L31.5723 16.20974C34.151 11.74872 38.9116 9 44.06 9h57.281c5.153 0 9.918 2.74872 12.488 7.20974z" clip-rule="evenodd" stroke="%231f2348" stroke-width="2" opacity=".2"/></svg>');
+    z-index: 1;
 }
 
-.transaction .total {
-    text-align: center;
+#account-details .label {
+    font-size: 3rem;
+    font-weight: 600;
+    line-height: 1;
+    margin-top: 3rem;
 }
 
-.transaction .total #value {
-    font-size: 6rem;
-    line-height: 8rem;
+#account-details .total {
+    margin-top: 2rem;
 }
 
-.transaction .total .nim-symbol {
+#account-details .total .balance {
+    font-size: 3rem;
+}
+
+#account-details .total .nim-symbol {
     margin-left: 1rem;
+    font-weight: 800;
 }
 
-.transaction .fee-section {
-    text-align: center;
+#account-details .wallet-label {
+    margin-top: 1.5rem;
+}
+
+#account-details .address {
+    width: 100%;
+    max-width: calc(28.25 * var(--nimiq-size, 8px));
+    font-family: 'Fira Mono', monospace;
     opacity: 0.5;
+    margin-top: 7rem;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    font-size: 3rem;
 }
 
-.transaction .data-section {
-    margin: 2rem 3rem 0;
+.chunk {
+    margin: calc(0.875 * var(--nimiq-size, 8px)) 0;
+    line-height: 1.11;
+    width: 33%;
+    height: 100%;
     text-align: center;
+    box-sizing: border-box;
+    text-transform: uppercase;
 }
 
-.layout-standard .nq-card-footer {
-    border-radius: 0;
+.space {
+    font-size: 0;
 }
 
-.page-footer {
-    padding: 1rem;
+.transaction .nq-card-body {
+    background-color: white;
 }
 
-.layout-standard ~ .page-footer {
-    background: white;
-    border-bottom-left-radius: 1rem;
-    border-bottom-right-radius: 1rem;
+.transaction .accounts {
+    flex-direction: row;
+    align-items: flex-start;
+    padding-bottom: 4rem;
 }
 
-/*
-#######################################
-#### C H E C K O U T   L A Y O U T ####
-#######################################
-*/
+.transaction .account .total,
```

Comment:
`.account .total` already has class `.display-none` that is dynamically removed when the balance is known.
This line here however prevents that it is ever visible.